lib/jemoji.rb

```diff
@@ -20,22 +16,18 @@ def src
         end
     end
 
-    def filter
-      @filter ||= HTML::Pipeline::EmojiFilter.new(nil, { :asset_root => src })
-    end
-
-    def generate(site)
-      site.posts.each { |doc| emojify doc } unless v3?
-      site.pages.each { |doc| emojify doc }
-      site.docs_to_write.each { |doc| emojify doc }
+    def matches(ext)
+      ext == ".html"
```

Comment:
This doesn't match `.md` or `.textile` or any other formats that convert into HTML. :/